### DISCUSSION TITLE
Sending two separate video streams with rtp-to-webrtc under two different ports

### DISCUSSION THREAD

**Discussion by Sean-Zeo** [pion/webrtc#2705] [Q&A] [answered]
Hi, could you provide an example of how to send two video feeds at once (port 5004 and port 5005 for example).
I'm having difficulty, thanks!

**Comment by Sean-Der** [comment]
Hi @Sean-Zeo 

Can you describe what you are doing a bit more? Are you sending video from your browser? Are you sending video from pion/webrtc? Where is the video coming from? Why do you mention ports, are you doing rtp forwarding?

thanks!

**Comment by Sean-Zeo** [comment]
Hi sir, I am using 'rtp-to-webrtc' example  I want to send both a video and audio track at the same time. :)
As separate inputs from separate ports.

**Reply by Sean-Der** [reply, marked as answer]
Create however many tracks you want, basically copy these lines [here](https://github.com/pion/webrtc/blob/master/examples/rtp-to-webrtc/main.go#L52-L60)

Create multiple listeners for each port you care about like [this](https://github.com/pion/webrtc/blob/master/examples/rtp-to-webrtc/main.go#L38-L50)

Start a new goroutine for each listener and start reading packets like [this](https://github.com/pion/webrtc/blob/master/examples/rtp-to-webrtc/main.go#L118-L133)

I would start a new repo under your github account and make the changes. If you get stuck drop the repo here and happy to help!
